### PR TITLE
Admin group UI

### PR DESCRIPTION
**Pull request by cgeorgilakis**

### CODE REVIEW

**Review by cgeorgilakis — COMMENTED** [review]
test

File: src/main/resources/theme/rciam-group/account/src/app/js/utils.js [hunk 1]
```diff
@@ -17,4 +17,19 @@ export function deepEqual(x, y) {
     ok(x).length === ok(y).length &&
       ok(x).every(key => deepEqual(x[key], y[key]))
   ) : (x === y);
+}
+
+export function isIntegerOrNumericString(value) {
+  if (Number.isInteger(value)) {
```
[the remark this answers is not in the document]
Comment (linathedog) [reply src/main/resources/theme/rciam-group/account/src/app/js/utils.js]: Done

File: src/main/resources/theme/rciam-group/account/resources/js/utils.js [hunk 2]
```diff
@@ -13,4 +13,18 @@ export function deepEqual(x, y) {
         ty = typeof y;
   return x && y && tx === 'object' && tx === ty ? ok(x).length === ok(y).length && ok(x).every(key => deepEqual(x[key], y[key])) : x === y;
 }
+export function isIntegerOrNumericString(value) {
```
[the remark this answers is not in the document]
Comment (linathedog) [reply src/main/resources/theme/rciam-group/account/resources/js/utils.js:16]: What is twice? The file?

[the remark this answers is not in the document]
Comment (cgeorgilakis) [reply src/main/resources/theme/rciam-group/account/resources/js/utils.js:16]: The file. The second is autogenerated?

**Pull request merged**